extensions/ql-vscode/src/pure/messages.ts

```diff
@@ -857,6 +856,21 @@ export interface RunUpgradeResult {
   finalSha: string;
 }
 
+export interface RegisterDatabaseParams {
+  databases: Dataset[];
+}
+
+export interface DeregisterDatabaseParams {
+  databases: Dataset[];
+}
+
+export type RegisterDatabaseResult = {
+  registeredDatabases: Dataset[];
+};
+
+export type DeregisterDatabaseResult = {
+  registeredDatabases: Dataset[];
+};
```

Comment:
These are plural in the Java code, e.g. `RegisterDatabasesParams`. Also update the internal copy of this file.

Reply:
Didn't know about that file...looks like it's a bit out of date. I can update.